lib/registries/addon/overview/styles.scss

```diff
@@ -78,3 +78,39 @@
     color: $primary-red;
     font-weight: bold;
 }
+
+.ResourcesWrapper {
+    display: inline-flex;
+    flex-direction: column;
+    background-color: #fff;
+    color: $primary-blue;
+    border-top: 1px solid #999;
+    padding-left: 8px;
+    width: 100%;
+}
+
+.ResourcesWrapper h3 {
+    font-weight: 700;
+    color: #333;
+}
+
+.ResourcesWrapper img {
+    height: 22.5px;
+    width: 22px;
+}
```

Comment:
Could you add a little margin-right to the img to get it to push the text label over a smidge?